lib/tasks/mastodon.rake

```diff
@@ -579,6 +578,10 @@ namespace :mastodon do
 
   private
 
+  def errors
+    @errors ||= []
+  end
```

Comment:
I found that slightly confusing. Anything against it being either a local variable, or explicitly an instance variable?

Reply:
Good catch, converted back to local var.

I think it may eventually be useful to convert to an i-var or otherwise get some sort of "accessor"-like behavior here so that we can pull apart this huge task method into smaller parts, but for purposes of this PR alone it's not needed yet.